diff --git a/js/ui/quickSettings.js b/js/ui/quickSettings.js
index d5e9ca88f..992628899 100644
--- a/js/ui/quickSettings.js
+++ b/js/ui/quickSettings.js
@@ -862,6 +862,7 @@ export const QuickSettingsMenu = class extends PopupMenu.PopupMenu {
         this.box.add_style_class_name('quick-settings');
         this.box.x_align = Clutter.ActorAlign.END;
         this.box.y_align = Clutter.ActorAlign.START;
+        this.box.y_expand = false;
         this.box.clip_to_allocation = true;
 
         this._innerBox = new St.BoxLayout({
